src/Microsoft.AspNetCore.Authentication.OAuth/OAuthHandler.cs

```diff
@@ -129,7 +129,15 @@ public OAuthHandler(HttpClient backchannel)
                 properties.StoreTokens(authTokens);
             }
 
-            return AuthenticateResult.Success(await CreateTicketAsync(identity, properties, tokens));
+            try
+            {
+                var ticket = await CreateTicketAsync(identity, properties, tokens);
+                return AuthenticateResult.Success(ticket);
+            }
+            catch(Exception ex)
```

Comment:
I don't think its an obvious expectation for developers to know to try/catch always around `HandleRemoteAuthenticateAsync`, seems much more defensive to just prevent exceptions from leaking ourselves.  Basically, the choice is between putting some doc comments saying uncaught exceptions in `HandleRemoteAuthenticateAsync` bypass the remote error handler, or just guarding against that possibility ourselves.